### PR TITLE
Update styling

### PR DESCRIPTION
**Pull request by dajanaheinze**

### CODE REVIEW

File: src/App.js [hunk 1]
```diff
@@ -4,18 +4,18 @@ import TutorialList from './components/TutorialList'
...
-    </PageLayout>
+    </AppLayout>
   )
 }
 
-const PageLayout = styled.div`
+const AppLayout = styled.div`
   display: grid;
-  grid-template-rows: 90px auto;
+  grid-template-rows: px auto;
```
Comment (mariegross) [inline src/App.js]: A number went missing :)

File: src/GlobalStyles.js [hunk 2]
```diff
@@ -9,9 +9,11 @@ export default createGlobalStyle`
...
     --secondary-background: #F2F2F2;
     --primary-text: #080808;
+    --secondary-text:#323331;
     --primary-border: #F2F2F2;
     --primary-shadow: #8c8d80;
     --secondary-shadow: #bebfae;
+    --third-shadow: #757670;
 
     --primary-black: #0D0D0D;
     --primary-anthrazit: #262626;
```
Comment (mariegross) [inline src/GlobalStyles.js]: I think it's best practice to stick to the same pattern when naming styling variables:
- maybe you find different expressions for `--primary-black` and `--primary-anthrazit` (that don't actually include the color)

File: src/components/AppHeader.js [hunk 3]
```diff
@@ -14,6 +14,9 @@ export default function AppHeader({ headline }) {
 const Header = styled.header`
   display: grid;
   background-color: var(--primary-background);
+  border-bottom-left-radius: 4px;
+  border-bottom-right-radius: 4px;
```
Comment (mariegross) [inline src/components/AppHeader.js]:
```suggestion
  border-radius: 0 0 4px 4px;
```

File: src/GlobalStyles.js [hunk 4]
```diff
@@ -9,9 +9,11 @@ export default createGlobalStyle`
     --primary-background: #A5A697;
     --secondary-background: #F2F2F2;
     --primary-text: #080808;
+    --secondary-text:#323331;
     --primary-border: #F2F2F2;
     --primary-shadow: #8c8d80;
     --secondary-shadow: #bebfae;
+    --third-shadow: #757670;
```
Comment (mariegross) [inline src/GlobalStyles.js]:
```suggestion
    --tertiary-shadow: #757670;
```

**Review by mariegross — COMMENTED** [review]
Added some small suggestions - looks great otherwise :)

File: src/GlobalStyles.js (hunk 2 above)
Comment (dajanaheinze) [reply src/GlobalStyles.js]: this mean the general Color. I thing about it.

**Pull request merged**